### PR TITLE
Feature/signup

### PR DESCRIPTION
**Pull request by antoniolopesg**
Tests will be added in the next task

### CODE REVIEW

File: packages/server/src/graphql/mutation.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { GraphQLObjectType } from 'graphql';
+import { userSignupMutation } from '../modules/user/mutations/userSignupMutation';
+
+export const mutation = new GraphQLObjectType({
+  name: 'Mutation',
+  description: 'Root Mutation',
+  fields: () => ({
+    signup: userSignupMutation,
```
Comment (daniloab) [inline packages/server/src/graphql/mutation.ts:8]: prefer `userSignup`

**Review by noghartt — APPROVED** [review]
LGTM

**Review by daniloab — APPROVED** [review]
cool

**Review by biantris — APPROVED** [review]

**Comment by biantris** [comment]
cool!

**Pull request merged**